src/main/java/org/gridsuite/filter/server/dto/VoltageLevelFilter.java

```diff
@@ -35,19 +37,26 @@ public EquipmentType getEquipmentType() {
     @Schema(description = "Countries")
     private SortedSet<String> countries;
 
+    @Schema(description = "Free properties")
+    private Map<String, Set<String>> freeProperties;
+
     @Schema(description = "Nominal voltage")
-    private NumericalFilter nominalVoltage;
+    private NumericalFilter          nominalVoltage;
```

Comment:
remove this spaces

Reply:
Good catch. I guess auto-formatter did this. I remove.

Reply:
Done

Reply:
Done